modules/trusted-firmware-m/nordic_nrf/include/RTE_Device.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef __RTE_DEVICE_H
+#define __RTE_DEVICE_H
+
+#include <autoconf.h>
+
+/* ARRAY_SIZE causes a conflict as it is defined both by TF-M and indirectly by devicetree.h */
+#undef ARRAY_SIZE
+#include <zephyr/devicetree.h>
+#include <nrf-pinctrl.h>
```

Comment:
Is this `zephyr/dt-bindings/pinctrl/nrf-pinctrl.h`? Perhaps I'm missing something, but is it actually needed to include it here?

Reply:
I think that is true, it's only needed for the vanilla TF-M RTE_Device.h headers, which don't have access to this header.
Will remove this include.